web/src/message_list_view.ts

```diff
@@ -394,14 +394,17 @@ function populate_group_from_message(
 
         const sub = sub_store.get(message.stream_id);
         let stream_id;
+        let user_can_resolve_topic;
         if (sub === undefined) {
             // Hack to handle unusual cases like the tutorial where
             // the streams used don't actually exist in the subs
             // module.  Ideally, we'd clean this up by making the
             // tutorial populate stream_settings_ui.ts "properly".
             stream_id = -1;
+            user_can_resolve_topic = settings_data.user_can_resolve_topic();
```

Comment:
I feel like this case could be simplified if `stream_data.can_resolve_topics(undefined)` returned the right thing.